ksqldb-functional-tests/src/test/resources/query-validation-tests/latest-offset-udaf.json

```diff
@@ -0,0 +1,41 @@
+{
+  "comments": [
+    "Tests covering the use of the LATEST_BY_OFFSET aggregate function"
+  ],
+  "tests": [
+    {
+      "name": "latest by offset",
+      "statements": [
+        "CREATE STREAM INPUT (ROWKEY BIGINT KEY, ID BIGINT, F0 INT, F1 BIGINT, F2 DOUBLE, F3 BOOLEAN, F4 STRING) WITH (kafka_topic='test_topic', value_format='JSON', key='ID');",
+        "CREATE TABLE OUTPUT AS SELECT ID, LATEST_BY_OFFSET(F0) AS L0, LATEST_BY_OFFSET(F1) AS L1, LATEST_BY_OFFSET(F2) AS L2, LATEST_BY_OFFSET(F3) AS L3, LATEST_BY_OFFSET(F4) AS L4 FROM INPUT GROUP BY ID;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": 0, "value": {"ID": 0, "F0": 12, "F1": 1000, "F2": 1.23, "F3": true, "F4": "foo"}},
+        {"topic": "test_topic", "key": 1, "value": {"ID": 1, "F0": 12, "F1": 1000, "F2": 1.23, "F3": true, "F4": "foo"}},
+        {"topic": "test_topic", "key": 0, "value": {"ID": 0, "F0": 21, "F1": 2000, "F2": 2.23, "F3": false, "F4": "bar"}},
+        {"topic": "test_topic", "key": 1, "value": {"ID": 1, "F0": 21, "F1": 2000, "F2": 2.23, "F3": false, "F4": "bar"}}
+      ],
```

Comment:
Can we use different input values for the two different keys please?  One could imagine this test passing even if the code didn't do things per-key...

Reply:
Ack